web/src/utils/helpers.ts

```diff
@@ -148,3 +153,44 @@ export function getNetExchange(
     ? round(zoneData.totalCo2NetExchange / 1e6 / 60) // in tCO₂eq/min
     : round(zoneData.totalImport - zoneData.totalExport);
 }
+
+/**
+ * Returns the net production of a zone
+ * @param zoneData - The zone data
+ * @returns The net production
+ */
+export function getNetProduction(
+  zoneData: ZoneDetail,
+  displayByEmissions: boolean
+): number {
+  return displayByEmissions
+    ? round(zoneData.totalCo2Production / 1e6 / 60) // in tCO₂eq/min
+    : round(zoneData.totalProduction);
+}
+
+/**
+ * Returns the net consumption of a zone
+ * @param zoneData - The zone data
+ * @returns The net consumption
+ */
+export function getNetConsumption(
+  zoneData: ZoneDetail,
+  displayByEmissions: boolean
+): number {
+  return displayByEmissions
+    ? round(
+        tonsPerHourToGramsPerMinute(
+          getTotalElectricity(zoneData, true, Mode.CONSUMPTION)
+        ) * 100
+      ) / 100 // in tCO₂eq/min
```

Comment:
This function was used this way in similar code in the EmissionChartTooltip.tsx file, so I supposed it was related with quarky floating point arithmetic.

Reply:
It just don't make much sense to me to multiply it by 100 then round it then divide it by 100 instead of just rounding it directly as the division and multiplication cancel each other out.

Reply:
The only thing useful about it is ensuring 2 decimal digits. It seems I confused the function round() with Math.round(), so it is unnecessary to do this steps. I'll change it as it is completely unnecessary using this function.